.github/workflows/update-packages.yml

```
@@ -35,6 +35,11 @@ jobs:
         run: |
           echo "solution=$(([io.fileinfo]$(Get-ChildItem -Path .\* -Include *.sln)).name)" | Out-File -FilePath $env:GITHUB_ENV -Append
 
+      - name: Setup .NET
+        uses: actions/setup-dotnet@v4
+        with:
+          dotnet-version: "8.0.x"
+      
```

yamllint

[error] trailing spaces

(trailing-spaces)